nix/overlays/required.nix

```diff
@@ -31,7 +31,7 @@ self: super: {
         inherit enableProfiling;
       };
     });
-    cardano-sl-wallet-static = justStaticExecutablesGitRev super.cardano-sl-wallet;
+    cardano-sl-wallet-static = justStaticExecutablesGitRev super.cardano-sl-wallet-new;
```

Comment:
Not sure if this is the iffy part of this

Reply:
I believe this can just go in favor of the line below:

```
 cardano-sl-wallet-new-static = justStaticExecutablesGitRev super.cardano-sl-wallet-new;
```

Reply:
Ooohh 🤦‍♂️